config.yml.default

```diff
@@ -1,131 +1,131 @@
+    offer_draw_moves: 10           # How many moves in a row the absolute value of the score has to be below the draw value.
+    offer_draw_pieces: 10          # Only if the pieces on board are less than or equal to this value, the bot offers/accepts draw.
 
   online_moves:
-    max_out_of_book_moves: 10 # Stop using online opening books after they don't have a move for 'max_out_of_book_moves' positions. Doesn't apply to the online endgame tablebases.
-    max_retries: 2           # The maximum amount of retries when getting an online move.
+    max_out_of_book_moves: 10      # Stop using online opening books after they don't have a move for 'max_out_of_book_moves' positions. Doesn't apply to the online endgame tablebases.
+    max_retries: 2                 # The maximum amount of retries when getting an online move.
     chessdb_book:
       enabled: false
       min_time: 20
-      move_quality: "good"   # One of "all", "good", "best".
-      min_depth: 20          # Only for move_quality: "best".
+      move_quality: "good"         # One of "all", "good", "best".
+      min_depth: 20                # Only for move_quality: "best".
     lichess_cloud_analysis:
       enabled: false
       min_time: 20
-      move_quality: "good"   # One of "good", "best".
-      max_score_difference: 50 # Only for move_quality: "good". The maximum score difference (in cp) between the best move and the other moves.
+      move_quality: "good"         # One of "good", "best".
+      max_score_difference: 50     # Only for move_quality: "good". The maximum score difference (in cp) between the best move and the other moves.
       min_depth: 20
       min_knodes: 0
     online_egtb:
-      enabled: false
+      enabled: true
       min_time: 20
       max_pieces: 7
-      source: "lichess"      # One of "lichess", "chessdb".
-      move_quality: "best"   # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
+      source: "lichess"            # One of "lichess", "chessdb".
+      move_quality: "best"         # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
 
-  lichess_bot_tbs:           # The tablebases list here will be read by lichess-bot, not the engine.
+  lichess_bot_tbs:                 # The tablebases list here will be read by lichess-bot, not the engine.
     syzygy:
       enabled: false
-      paths:
+      paths:                       # Paths to Syzygy endgame tablebases.
         - "engines/syzygy"
-      max_pieces: 7
-      move_quality: "best"   # One of "good", "best", "suggest" (it takes all the "good" moves and tells the engine to only consider these; will move instantly if there is only 1 "good" move).
+      max_pieces: 5                # Maximum number of pieces in the endgame tablebase.
```

Comment:
I think 7 is better, so that all syzygy TBs can be used by default.

Reply:
Changed.